airflow/contrib/operators/gcp_compute_operator.py

```diff
@@ -24,16 +24,19 @@
 from airflow.contrib.hooks.gcp_compute_hook import GceHook
 from airflow.contrib.utils.gcp_field_sanitizer import GcpBodyFieldSanitizer
 from airflow.contrib.utils.gcp_field_validator import GcpBodyFieldValidator
+from airflow.contrib.utils.input_validator import InputValidationMixin
 from airflow.models import BaseOperator
 from airflow.utils.decorators import apply_defaults
 from json_merge_patch import merge
 
 
-class GceBaseOperator(BaseOperator):
+class GceBaseOperator(BaseOperator, InputValidationMixin):
     """
     Abstract base operator for Google Compute Engine operators to inherit from.
     """
 
+    REQUIRED_ATTRIBUTES = ['project_id', 'zone', 'resource_id']
```

Comment:
This PR by @potiuk introduces non-obligatory `project_id` parameter. We could probably already change it here to avoid conflicts in the future - so no required `project_id` attribute (everywhere in REQUIRED_ATTRIBUTES).